src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -15,6 +15,8 @@
 package org.candlepin.controller;
 
 import org.candlepin.audit.EventSink;
+import org.candlepin.cache.ConsumerCertContent;
+import org.candlepin.cache.ConsumerCertificateContentCache;
```

Comment:
For the same reason as the config property name: the 'consumer' part of the name is redundant. can we make it more clear these are for anonymous certificate content? something like `AnonymousCertContent` and `AnonymousCertContentCache`?

Reply:
Updated